nuxeo-runtime/nuxeo-stream/pom.xml

```diff
@@ -18,6 +18,16 @@
       <groupId>net.openhft</groupId>
       <artifactId>chronicle-queue</artifactId>
     </dependency>
+    <dependency>
+      <groupId>net.openhft</groupId>
+      <artifactId>chronicle-threads</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>net.openhft</groupId>
+          <artifactId>affinity</artifactId>
```

Comment:
Please add a comment explaining why we exclude it.  Also it's probably better to put this exclusion in the toplevel POM.

Reply:
Explained.
In the toplevel POM only chronicle BOM is declared, it seems more relevant excluded down there

Reply:
it appears maven eclipse:eclipse does not support the case where the bom is imported and a dependency from it is declared alongside, I moved it back to stream module since the chronicle dependency will not be used elsewhere